diesel_compile_tests/tests/fail/delete_statement_does_not_support_returning_methods_on_sqlite.rs

```diff
@@ -1,22 +0,0 @@
-extern crate diesel;
```

Comment:
I'm not sure if we should remove this tests. We don't enable the new feature as part of the `diesel_compile_tests` build, right?

Reply:
yes, right. done